Modules/main.c

```diff
@@ -1714,6 +1731,27 @@ cmdline_get_env_flags(_Py_CommandLineDetails *cmdline)
 }
 
 
+/* Set global variable variables from environment variables */
+void
+_Py_Initialize_ReadEnvVars(void)
```

Comment:
Something like `_Py_Initialize_ReadEnvVarsNoAlloc` might be a better name here, as this doesn't read *all* the env vars, only the ones that it can process without needing to allocate any memory to store the results.

With the current name, it gives the impression it reads all the environment variables, when it actually skips over the more complex ones like `PYTHONPATH`.

Reply:
function renamed